misk-redis/src/test/kotlin/misk/redis/AbstractRedisTest.kt

```diff
@@ -373,8 +374,8 @@ abstract class AbstractRedisTest {
 
   @Test fun hrandFieldUnsupportedCount() {
     // The Redis HRANDFIELD specification dictates different behaviour when count is negative.
-    // This behaviour cannot be adhered to by our implementation until Jedis fixes this bug:
-    // https://github.com/redis/jedis/issues/3017
+    // This behaviour cannot be adhered to by our implementation until redis fixes this bug:
+    // https://github.com/redis/redis/issues/3017
```

Comment:
This points to the wrong PR now?

Reply:
Ahh.. At some point I did Find & Replace.. I missed reverting this change. Will do that.